src/metadata/ActionMetadata.ts

```diff
@@ -256,7 +256,7 @@ export class ActionMetadata {
     
     static appendBaseRouteToRegexpRoute(route: RegExp, baseRoute: string) {
         if (!baseRoute || baseRoute === "") return route;
-        const fullPath = baseRoute.replace("\/", "\\\\/") + route.toString().substr(1);
+        const fullPath = `^${baseRoute}${route.toString().substr(1)}?$`;
```

Comment:
I don't remember why `.replace("\/", "\\\\/")` is there, maybe its 	
excessive. (tip of myself: comment strange peace of codes 😆 )